documentation/userguide/docs/shares.md

```diff
@@ -197,27 +200,93 @@ we receive an error. Once we have revoked access to all items we can delete the
+2. Grant access to the datashare to the consumer namespace (same account) or to the consumer AWS account (cross account)
+3. (If cross-account share) Authorize and associate datashare with the target namespace
+4. In the target namespace, create local database for the datashare and grant permissions to the principal Redshift role.
+5. In the target namespace, create external schema in local database and grant usage permissions to the principal Redshift role.
+6. For the local database and for the external schema, grant select access to the requested table to the principal Redshift role.
 
 ## **Consume shared data**
-Data.all tables are Glue tables shared using AWS Lake Formation, therefore any service that reads Glue tables and integrates
-with Lake Formation is able to consume the data. Permissions are granted to the team role or the consumption role that 
-has been specified in the request.
+Knowing what we know form the previous section we can now define some ways of consuming the shared data for each type of shareable item.
+
+### S3 Bucket sharing
+For S3 bucket sharing, IAM policies, S3 bucket policies, and KMS Key policies (if applicable) are updated to enable sharing of the S3 Bucket resource.
+Therefore, we can use S3 API calls to access the data referring the Bucket directly. We need to assume or use the credentials
+of the principal IAM role used in the share request (team IAM role or consumption IAM role).
 
-For the case of folders, the underlying sharing mechanism used is S3 Access Points. You can read data inside a prefix using 
-the IAM role of the requester (same as with tables) and executing get calls to the S3 access point.
+Here is an example using the AWS CLI:
 
-For example:
 ```json
- aws s3 ls arn:aws:s3:<SOURCE_REGION>:<SOURCE_AWSACCOUNTID>:accesspoint/<DATASETURI>-<REQUESTER-TEAM>/folder2/
+ aws s3 ls s3://<BUCKET_NAME>
 ```
 
-For S3 bucket sharing, IAM policies, S3 bucket policies, and KMS Key policies (if applicable) are updated to enable sharing of the S3 Bucket resource.
+### Glue Table sharing
+
+Glue tables are shared using AWS Lake Formation, therefore any service that reads Glue tables and integrates
+with Lake Formation is able to consume the data.
+
+We need to assume or use the credentials
+of the principal IAM role used in the share request (team IAM role or consumption IAM role).
 
-For example, access to the bucket would be similar to:
+### S3 Prefix sharing (Folders)
+For the case of folders, the underlying sharing mechanism used is S3 Access Points. You can read data inside a prefix 
+executing API calls to the S3 access point.
+
+We need to assume or use the credentials
+of the principal IAM role used in the share request (team IAM role or consumption IAM role).
+
+For example, we could use the AWS CLI with the following access point:
 ```json
- aws s3 ls s3://<BUCKET_NAME>
+ aws s3 ls arn:aws:s3:<SOURCE_REGION>:<SOURCE_AWSACCOUNTID>:accesspoint/<DATASETURI>-<REQUESTER-TEAM>/folder2/
```

Comment:
should we give example like `folder2` here or keep generic like `<FOLDER_NAME>`

Reply:
I think I prefer the folder name